.github/workflows/jira.yaml

```diff
@@ -25,9 +25,9 @@ jobs:
       id: preprocess
       run: |
         if [[ "${{ github.event_name }}" == "pull_request_target" ]]; then
-          echo "::set-output name=type::PR"
+          echo "type=PR" >> "$GITHUB_OUTPUT"
```

Comment:
`set-output` is deprecated. This was detected by actionlint.

Reply:
Nice! I've seen the deprecation notices for set output on other repos as well, like TFVP. Good to know this is how we resolve it 😄